lib/utils/cert/selfsigned.go

```diff
@@ -87,6 +87,10 @@ func GenerateSelfSignedCert(hostNames []string) (*Credentials, error) {
 	if ips != nil {
```

Comment:
Nit: It'd be tempted to rename `ips` to `loopbackIPs` or similar - just so its clearly distinct from `ipAddresses` in purpose.